packages/optimizer/README.md

```diff
@@ -23,20 +23,36 @@ The performance optimizations can improve page rendering times by up to 50%. You
 
 ## Usage
```

Comment:
@patrickkettner I've tried to document the two new modes. Wdyt? 

Fast and Full are placeholders...